imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -172,10 +174,11 @@ def find_sweep_starts(packets: xr.Dataset) -> np.ndarray:
         & ione[9:-1]
         & ione[10:]
     )
-    return np.where(valid)[0]
+    indices = np.where(valid)[0]
+    return indices
```

Comment:
This one can be reverted too.
```suggestion
    return np.where(valid)[0]
```